packages/flutter/test/painting/text_painter_test.dart

```diff
@@ -1726,6 +1744,36 @@ void main() {
       areCreateAndDispose,
     );
   });
+
+  testWidgets('TextPainter computeWidth should return the longest width for the longest word with Material font', (WidgetTester tester) async {
```

Comment:
I'm not sure what this is verifying? `getWidestWord` seems to be a method defined in the test file?

Reply:
Also I'm not sure the material icons font has latin glyphs. If not then all these characters will be replaced by the `.notDef` glyph so `IIIII` and `WWWW` would just have the same width.